demo/nuxt.config.js

```diff
@@ -1,13 +1,11 @@
 import TsconfigPathsPlugin from 'tsconfig-paths-webpack-plugin'
 import coreModuleDist from '../src/module'
 
-const API_URL_BROWSER = process.env.API_URL_BROWSER || 'https://localhost:8443'
```

Comment:
I removed this because it's not used anywhere and brings confusion, we can revert ofc.

Reply:
Cool, yes only needed in the template application which runs in docker because browser and server URLs are different